src/main/scala/com/cognite/sdk/scala/common/dataTypes.scala

```diff
@@ -22,12 +22,21 @@ final case class ItemsWithRecursiveAndIgnoreUnknownIds(
     ignoreUnknownIds: Boolean
 )
 
+trait CdfSdkException {
```

Comment:
If we change it so we don't need `CdfSdkIllegalArgumentException` we don't really need this, we can just catch both `SdkException` and `CdpApiException` and only let those through.